Dashboard/src/components/Github.js

```diff
@@ -3,17 +3,17 @@ import Panel from './common/Panel';
 import Container from './common/Container';
 import github from './assets/github.svg';
 import moment from 'moment';
-const COMPONENT_NAME = "github";
+const COMPONENT_NAME = "github"
 class Github extends Container {
   constructor(props) {
     super(props, COMPONENT_NAME);
   }
 
   render() {
     return (
-      <div>
+      <div className={this.constructor.name.toLocaleLowerCase()+ '-panel panel'}>
```

Comment:
The purpose of them is to identify them in the browser. This makes it easier when debugging and looking at the source code. Initially the div had no way of identifying it. This can be applied on all future panels and styling can be used to give panels unique customization. I found that the reason it serves is to wrap the logic it contains beneath the panel component.

Reply:
In that case I recommend we add it to the Panel component itself to encapsulate this logic.

Reply:
We could write a template for the Panel object in which we pass a closure to the constructor and implement it in the panel object being created. The panel class as I understood it is a template for all panels.

Reply:
I was thinking of just wrapping `render` with `<div className=`${props.title}-panel panel`>`

Reply:
fixed